src/Tools/babylon.tools.ts

```diff
@@ -341,6 +341,22 @@
             }
         }
 
+        //returns a downloadable url to a file content.
+        public static FileAsURL(content:string): string {
+            var fileBlob;
+            try {
+                fileBlob = new Blob([content]);
+            } catch(e) {
+                var blobBuilder = window.BlobBuilder || window.MozBlobBuilder || window.WebKitBlobBuilder;
```

Comment:
BlobBuilder is obsolete and should not be used, https://developer.mozilla.org/en-US/docs/Web/API/BlobBuilder
Since this is a WebGL framework, only the newest browsers are supported, and they all support blob - http://caniuse.com/#search=blob